common/primitives/constants.go

```diff
@@ -50,3 +50,7 @@ const (
 const (
 	DefaultHistoryMaxAutoResetPoints = 20
 )
+
+const (
+	ScheduleWorkflowIDPrefix = "temporal-sys-scheduler:"
```

Comment:
There's already a constant in `service/worker/scheduler/workflow.go`.
Can you avoid duplication?

Reply:
That was the constant being used initially, but it causes a circular reference. Per recommendation from @dnr , it was defined in primitives.

Reply:
It should be _moved_ to primitives, not duplicated there. Can you point the other uses to the new one and remove the old one?